java/arcs/core/storage/DirectStoreMuxer.kt

```diff
@@ -75,12 +75,12 @@ class DirectStoreMuxer<Data : CrdtData, Op : CrdtOperation, T>(
 
     /** Calls [idle] on all existing contained stores and waits for their completion. */
     suspend fun idle() = storeMutex.withLock {
-        stores.values.map {
-            withContext(coroutineContext) {
-                launch { it.store.idle() }
-            }
-        }.joinAll()
-    }
+        stores.values.toList()
+    }.map {
+        withContext(coroutineContext) {
+            launch { it.store.idle() }
```

Comment:
Is there any benefit to launching these in parallel if the goal is just to wait for all to complete?

Reply:
It depends on the overhead of `store.idle()`.
If the overhead of each `store.idle()` is bigger than an `launch` overhead, the launch here can help more or less.
If not, the launch here would hurt instead.

Just take a look at the implementation of `store.idle()`, it does nothing at this moment...
So we can remove the multiple launches to get some time back.
Nice find!

Reply:
btw, the PR improves writes for all "persistent" handles (I knew we changed to memdb for pexxx) a lot from 1+ second to around 30 ms (100 participants).

Reply:
Removed launches and added comments.